src/renderer/widgets/Image.cpp

```diff
@@ -190,14 +190,6 @@ bool CImage::draw(const SRenderData& data) {
     return data.opacity < 1.0;
 }
 
-static void onAssetCallbackTimer(std::shared_ptr<CTimer> self, void* data) {
```

Comment:
why is this gone

Reply:
Cause i also removed that from `Label` in #384
I thought we don't need it any more, since we set `needsFrame = true`, when `frameCallback` is present in `CSessionLockSurface::render`.

But maybe readd it just to be safe and maybe fix #390?

Reply:
I just dont think it belongs in this mr

Reply:
Fair